elastic/kibana
# [EPM] Include both kibana and elasticsearch assets in package information

## Summary

Fixes https://github.com/elastic/kibana/issues/85968

It was probably an oversight that `elasticsearch` assets weren't included in the original implementation. Half of the changes are uncommenting lines that have been commented out before.

### How to test this

Verify that https://github.com/elastic/kibana/issues/85968 has been fixed

Find an integration that contains both elasticsearch and kibana assets. Navigate to that integration's detail page under Fleet->Integrations (I used Nginx for this). Observe that both kind of assets are listed on the left side:
![image](https://user-images.githubusercontent.com/189073/104440754-06f1de80-5593-11eb-8739-cbb6a8ac1546.png)

On the integration's detail page, navigate to the Settings tab and click the "Install $INTEGRATION assets" button. Verify that the number of assets to be installed is correct:

![image](https://user-images.githubusercontent.com/189073/104440933-46202f80-5593-11eb-99dc-6d5669d4c652.png)
